star-wasm/src/lib.rs

```diff
@@ -1,7 +1,10 @@
-// Work around clippy warning from wasm_bidgen 0.2.79.
-// Can be removed after bumping to a release containing the upstream
-// fix in https://github.com/rustwasm/wasm-bindgen/pull/2778
-#![allow(clippy::unused_unit)]
+//! WebAssmbly bindings for the STAR protocol
```

Comment:
Just a typo here: 'WebAssembly".

Reply:
Oops! Fixed in e2f027f93c7a91f491ea367848674cee32a64a53.